CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# v0.0.0
```

Comment:
Does this changelog gnerate automatically? If not I would recommend using semantic commits, and automatic tag versioning with a simple gradle custom task.

Reply:
I added a task to do it.